protocols/grpc/agent.proto

```diff
@@ -37,9 +37,13 @@ service AgentService {
 	rpc TtyWinResize(TtyWinResizeRequest) returns (google.protobuf.Empty);
 
 	// networking
+	// Add interface, the interface should be removed when failed.
 	rpc AddInterface(AddInterfaceRequest) returns (google.protobuf.Empty);
```

Comment:
I was thinking one more thing for this. Do you allow `AddInterfaceRequest` has some empty field?  I would suppose we do.

For example, if AddInterface doesn't specify a MTU value, agent should set a default MTU for it. 

Then `AddInterface` should also return an `Interface` as response to tell runtime the final state of interface, to let user know which value is chosen by agent.